package.json

```diff
@@ -85,37 +85,6 @@
     "*.{js,json,md,mjs,ts,yml}": "prettier --write"
   },
   "prettier": "@stylelint/prettier-config",
-  "eslintConfig": {
-    "parserOptions": {
-      "ecmaVersion": 2023
-    },
-    "extends": [
-      "stylelint",
-      "stylelint/jest"
-    ],
-    "overrides": [
-      {
-        "files": [
-          "**/*.mjs"
-        ],
-        "plugins": [
-          "eslint-plugin-import"
-        ],
-        "rules": {
-          "import/extensions": [
-            "error",
-            "ignorePackages"
-          ]
-        }
-      }
-    ],
-    "globals": {
-      "testRule": "readonly",
-      "testRuleConfigs": "readonly"
-    },
-    "reportUnusedDisableDirectives": true,
-    "root": true
```

Comment:
Is `"root": true` not necessary anymore?

Reply:
It [no longer exists](https://eslint.org/docs/latest/use/configure/migration-guide#additional-changes) for flat configs.